emails/views.py

```diff
@@ -1634,6 +1645,22 @@ def _handle_complaint(message_json: AWS_SNSMessageJSON) -> HttpResponse:
         profile.auto_block_spam = True
         profile.save()
 
+    for destination_address in message_json.get("mail", {}).get("destination", []):
+        try:
+            address = _get_address(destination_address, False)
```

Comment:
Note: I changed the function call signature on this to accept a new `bool` parameter to control if the function will/won't create the necessary `DomainAddress` object.

There was enough logic in the `_get_address` function that I didn't want to re-write it. I considered re-factoring into 2 separate `_get_address` and `_get_or_create_address` functions, but that would require changing [all 17 calls `_get_address`](https://github.com/search?q=repo%3Amozilla%2Ffx-private-relay+_get_address&type=code), which seemed like a more disruptive change.